src/zenml/zen_server/routers/workspaces_endpoints.py

```diff
@@ -69,15 +69,25 @@
     zen_store,
 )
 
+# This is a workaround to slowly deprecate the projects routes. This along with
+#  all the decorators using it, can be removed in a few releases.
+PROJECTS = "/projects"
+
+
 router = APIRouter(
-    prefix=API + VERSION_1 + WORKSPACES,
+    prefix=API + VERSION_1,
     tags=["workspaces"],
     responses={401: error_response},
 )
 
 
 @router.get(
-    "",
+    WORKSPACES,
+    response_model=Page[WorkspaceResponseModel],
+    responses={401: error_response, 404: error_response, 422: error_response},
+)
+@router.get(
+    PROJECTS,
```

Comment:
putting two `@router...` decorators on top of each other simply uses this method for two endpoints? Awesome if it works like this, but would recommend testing manually once to make sure.

Reply:
https://github.com/tiangolo/fastapi/discussions/4299 

Got it there, it works in my tests (dashboard also still works, and it heavily relies on the project routes)

Reply:
Cool. In the answer you linked I saw you can even do `@router.post(..., deprecated=True)`, should we also do this for all the project endpoints?

Reply:
lets do it

Reply:
🔥